protocol_connect.go

```diff
@@ -53,6 +53,7 @@ const (
 	connectUnaryMessageQueryParameter     = "msg"
 	connectUnaryBase64QueryParameter      = "b64"
 	connectUnaryCompressionQueryParameter = "cmp"
+	connectUnaryTimeoutQueryParameter     = "t"
 	connectUnaryAPIQueryParameter         = "api"
```

Comment:
Reviewed this API briefly with Peter. He'd prefer longer query parameter names: "encoding", "message", "base64", "compression", and "timeout". I think he'd probably prefer "connect=v1" instead of "api=connectv1" too. We've got enough params here that I tend to agree - we've got kilobytes of space to work with, and making the overall URL more readable to people who don't know the protocol is valuable.